drivers/timer/stm32_lptim_timer.c

```diff
@@ -185,14 +185,8 @@ void z_clock_set_timeout(s32_t ticks, bool idle)
 		return;
 	}
 
-	/* ARROK bit validates previous write operation to ARR register */
-	while (LL_LPTIM_IsActiveFlag_ARROK(LPTIM1) == 0) {
-	}
-	LL_LPTIM_ClearFlag_ARROK(LPTIM1);
-
```

Comment:
This part of the change is not explained

Reply:
N/A  anymore, now merged in the #25940